source/_docs/ecosystem/certificates/lets_encrypt.markdown

```diff
@@ -474,37 +454,12 @@ $ crontab -e
  
  
 #### Option 2:
```

Comment:
Did you remove the whole section on purpose?

Reply:
Yes, because the section is entirely about adding homeassistant to sudoers.

Reply:
Not according to the diff. `Option 2:` covers the renewal process through an automation rule.